react/src/lib/components/DeckGLMap/layers/layersDefaultProps.ts

```diff
@@ -37,6 +37,27 @@ export const layersDefaultProps: Record<string, unknown> = {
             step: 0,
         },
     },
+    MapLayer: {
+        "@@type": "MapLayer",
+        name: "Map",
+        id: "map3d-layer-float32",
+        pickable: true,
+        visible: false,
```

Comment:
Odd to have default visibility false. Normally if you add a layer you intend to see it.